docker/docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: '3'
+services:
+  whereabouts-database:
+    container_name: whereabouts-database
+    image: postgres:11-alpine
+    environment:
+      POSTGRES_USER: wa
+      POSTGRES_PASSWORD: wa
+      POSTGRES_DB: whereabouts
+    volumes:
+      - db-data:/var/lib/postgresql/data
```

Comment:
I have the same issue with PostgreSQL not having role `wa`. I think is has to do with the name of the volume, probably should be a more unique name, probably collides with the gb-backend one.
See `docker volume ls` for the list of volumes.

Reply:
you're right, after changing the name of the volume and removing the previous one it works!